client/stylesheets/_toCat.scss

```diff
@@ -833,6 +833,7 @@ input[type="search"]::-webkit-search-results-decoration { display: none; }
     text-align: center;
     font-size: 4rem;
     color: $color-spacegrey;
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces